Demo/Demo/Kingfisher-Demo/ViewController.swift

```diff
@@ -95,3 +95,15 @@ extension ViewController: UICollectionViewDataSourcePrefetching {
         ImagePrefetcher(urls: urls).start()
     }
 }
+
+
+// Inspired by: https://fdp.io/blog/2018/03/22/supporting-compactmap-in-swift-4/
```

Comment:
👋 It's funny to see my blog post on a library I use 😄

Reply:
XD